MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -82,13 +101,19 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      - parameter imageURL: The URL to an image representation of this component.
      - parameter abbreviation: An abbreviated representation of `text`.
      - parameter abbreviationPriority: The priority for which the component should be abbreviated.
+     - parameter directions: The possibile directions to go from a lane component.
+     - parameter active: An active flag to indicate that the upcoming maneuver can be completed with a lane component.
      */
-    @objc public init(type: VisualInstructionComponentType, text: String?, imageURL: URL?, abbreviation: String?, abbreviationPriority: Int) {
+    @objc public init(type: VisualInstructionComponentType, text: String?, imageURL: URL?, abbreviation: String?, abbreviationPriority: Int, directions: [String]? = nil, active: Bool = false) {
         self.text = text
         self.imageURL = imageURL
         self.type = type
         self.abbreviation = abbreviation
         self.abbreviationPriority = abbreviationPriority
+        if let directions = directions {
+            self.directions = directions.map { ManeuverDirection(description: $0) ?? .none }
```

Comment:
I don't think `directions ` is an array of ManeuverDirctions. Try using the `LaneIndication` initializer here.

Reply:
@bsudekum yes (according to the recently published api-docs), when `type == .lane` we do have an array of directions. The suggested `LaneIndication` indicator could be a computed property as `type` is defined when initialized.

Reply:
@vincethecoder I'm saying that internally in this library, these are LaneIndications and not ManeuverDirections.